src/v2/Components/Footer.tsx

```diff
@@ -59,12 +62,65 @@ export const SmallFooter = (props: Props) => (
   />
 )
 
+const DownloadAppBanner = () => {
+  return (
+    <Flex
+      flexDirection="row"
+      justifyContent="center"
+      borderBottom={`1px solid ${color("black10")}`}
+      mb={3}
+      overflow="hidden"
+      id="download-app-banner"
+    >
+      <Flex
+        alignItems="center"
```

Comment:
Out of curiosity, are these order dependent? Or is this something we can alphabetize for helping with readability?

Reply:
These aren't order dependent and could definitely be alphabetized. We previously had a VS code extension that automatically alphabetized for us but found it created a lot of noise since it would reformat a lot of code that wasn't relevant for the PR.